### PR TITLE
Some more docs updates

### PR DESCRIPTION
**Pull request by jcyuan2020**
Pull request overview
---------------------
 A few changes on the documentation/testfiles:
 - Corrected a few equations used in the HAMT model which could easily cause some confusions, spotted while working on helpdesk ticket 15762;
- Updated a few test file introduction test, spotted while working on a fix pr before;
- Some other minor documentation changes and corrections scattered around.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex [hunk 1]
```diff
@@ -25,7 +25,7 @@ \subsection{HAMT Nomenclature}\label{hamt-nomenclature}
 C & J/kgC & Specific heat capacity of dry material \tabularnewline
 $C_i^h$ & J/C & Heat Capacitance of cell i \tabularnewline
 $C_i^w$ & kg & Moisture Capacitance of cell i \tabularnewline
-$c_{w}$ & J/KgC & Specific heat capacity of water (= 4,180~J/kg$^{o}$C@ 20$^{o}$C) \tabularnewline
+$c^{w}$ & J/KgC & Specific heat capacity of water (= 4,180~J/kg$^{\circ}$C@ 20$^{\circ}$C) \tabularnewline
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex:28]: Is this right?  It's c^w?  :shrug:

File: doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex [hunk 2]
```diff
@@ -34,24 +34,24 @@ \subsection{HAMT Nomenclature}\label{hamt-nomenclature}
...
+${q_i^{adds}}$ & W & Heat from additional Sources \tabularnewline
+${q_i^v}$ & W & Heat due to Vaporisation \tabularnewline
 RH & \% & Relative humidity \tabularnewline
 $R_{ij}^h$ & C/W & Heat Resistance between cells i and j \tabularnewline
 $R_{ij}^v$ & sPa/kg & Vapor Resistance between cells i and j \tabularnewline
-$^{R_{ij}^w}$ & s/kg & Liquid Moisture Resistance between cells i and j \tabularnewline
+$R_{ij}^w$ & s/kg & Liquid Moisture Resistance between cells i and j \tabularnewline
 $t$ & s & Time \tabularnewline
-T & °C & Temperature \tabularnewline
+T & $^\circ$C & Temperature \tabularnewline
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex:45]: :+1:

File: testfiles/GSHP-GLHE-CalcGFunctions-cpg.idf [hunk 3]
```diff
@@ -3,10 +3,9 @@
 !-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
 !-      Use '!' comments if they need to be retained when using the IDFEditor.
 !-   ===========  ALL OBJECTS IN CLASS: VERSION ===========
-! SlabInGradeWithInsulation.idf
+! GSHP-GLHE-CalcGFunctions-cpg.idf
```
Comment (Myoldmopar) [inline testfiles/GSHP-GLHE-CalcGFunctions-cpg.idf:6]: Thanks for tidying these up.

**Review by Myoldmopar — COMMENTED** [review]
Are you expecting this to be the full set of changes for this PR?  The only thing I thought was curious was the cw thing.  If that's actually correct then this can merge anytime.

File: doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex (hunk 1 above)
Comment (jcyuan2020) [reply doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex:28]: That's what's used in the equation. We can change either here to c^w or a few place in the equations to c_w.

Comment (Myoldmopar) [reply doc/engineering-reference/src/surface-heat-balance-manager-processes/combined-heat-and-moisture-transfer-hamt.tex:28]: :shrug: OK, thanks, we'll go with it.

**Comment by Myoldmopar** [comment]
Dropping this in, thanks @jcyuan2020

**Pull request merged**